resources/analysisTools/copyNumberEstimationWorkflow/phasing_X.sh

```diff
@@ -3,6 +3,13 @@
 # Copyright (c) 2017 The ACEseq workflow developers.
 # Distributed under the MIT License (license terms are at https://www.github.com/eilslabs/ACEseqWorkflow/LICENSE.txt).
 
+dieWith() {
```

Comment:
Pitty to repeat this in every file. Maybe source a library file? If you want, you could 

  1. Make a `resources/analysisTools/copyNumberEstimationWorkflow/bashLib.sh` script (similar in alignment workflow) and put the code in there. Make it executable (Roddy tools need to be executable).
  2. declare a `bashLib` tool in the XML
   ```xml
   <tool name="bashLib" value="bashLib.sh" basepath="copyNumberEstimationWorkflow"/>
   ```
  3. Then in all scripts that need it do `source "$TOOL_BASH_LIB"`

Thus you can centralize your bash code.